homework2/tests/test_hw1.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from homework2.hw1 import *
+
+
+def test_positive_case_get_longest_diverse_words():
```

Comment:
no need to use "positive_case" in each test functino name. it should obvious that it's positive by name itself